SensioLabs/Security/SecurityChecker.php

```diff
@@ -66,4 +75,56 @@ public function getCrawler()
     {
         return $this->crawler;
     }
+
+    /**
+     * Retrieves list of exclusions from external config file.
+     *
+     * @param string $configFile The path to the securityChecker.json file
+     *
+     * @return array An array of vulnerabilities to exclude
+     */
+    private function getSecurityExclusions($configFile)
+    {
+        try {
+            $config = json_decode(file_get_contents($configFile), true);
+
+            if ($config['exclusions'] && is_array($config['exclusions'])) {
+                return $config['exclusions'];
+            }
+        } catch (\Exception $e) {
+            throw new RuntimeException('Config file does not contain valid json.');
```

Comment:
`json_decode()` doesn't throw any exception for invalid JSON, instead will return `null` - I'd throw on null if `$config === null`

Reply:
Good point.. Been doing too much javascript!

Reply:
@jamie-infinity https://github.com/graemechapman/security-checker/pull/1/commits/de58eed620db5aac07959dc2930dbeab7855e944